docs/resources/job.md

```diff
@@ -346,6 +347,12 @@ The `condition_task` specifies a condition with an outcome that can be used to c
 
 This task does not require a cluster to execute and does not support retries or notifications.
 
+### for_each_task Configuration Block
+
+* `concurrency` - (Optional) Controls the number of active iteration task runs. Default is 100 (maximal value).
```

Comment:
@geekswaroop we changed the default to 20. It's OK if it is only fixed in the next regular Terraform release.

Reply:
I think the doc [fix](https://github.com/databricks/terraform-provider-databricks/pull/3271) is part of this release.

Reply:
Awesome, well done!